src/app/SharpRaven/Data/ExceptionFrame.cs

```diff
@@ -219,5 +223,11 @@ public override string ToString()
 
             return sb.ToString();
         }
+
+        private static bool IsSystemModuleName(string moduleName)
+        {
+            return !string.IsNullOrEmpty(moduleName) &&
+                moduleName.StartsWith("System.", System.StringComparison.Ordinal);
```

Comment:
Perhaps include `Microsoft.` and possible other prefixes known to be external? This should probably be configurable in the client as well, so people can adjust this as they see fit.